velox/docs/functions/spark/string.rst

```diff
@@ -89,6 +89,15 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns the length of ``string`` in characters.
 
+.. spark:function:: levenshtein(string, string[, threshold]) -> integer
+
+    Returns the `Levenshtein distance <https://en.wikipedia.org/wiki/Levenshtein_distance>`_ between the two given strings. If
+    ``threshold`` is set and distance more than it, return -1. ::
```

Comment:
nit: suggestion on the documentation: distance more than it -> distance exceeds it